.drone.yml

```diff
@@ -0,0 +1,53 @@
+---
+kind: pipeline
+name: build
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: gomod
+  pull: always
+  image: golang:1.12
+  commands:
+  - go mod vendor
+  - git diff --exit-code
+  environment:
+    CGO_ENABLED: 0
+    GO111MODULE: on
+  when:
+    event:
+      exclude:
+      - tag
+
+- name: build
+  pull: always
+  image: golang:1.12
+  commands:
+  - make build
+  - make test
+  environment:
+    CGO_ENABLED: 0
+    GO111MODULE: on
+  when:
+    event:
+      exclude:
+      - tag
+
+- name: generate
+  pull: always
+  image: golang:1.12
+  commands:
+  - make check-license
+  - make generate
+  - git diff --exit-code
+  environment:
+    CGO_ENABLED: 0
+    GO111MODULE: on
+  when:
+    event:
+      exclude:
+      - tag
+
+...
```

Comment:
is this valid yaml?

Reply:
Yes. Search for `std.manifestYamlStream(value)` on https://jsonnet.org/ref/stdlib.html. It's terminator for a YAML document stream. Not really necessary, but it gets generated by `drone jsonnet` and doesn't really hurt.